hazelcast-sql/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -817,6 +817,22 @@ SqlNode SqlExplainStatement() :
     }
 }
 
+SqlNode SqlAnalyzeStatement() :
+{
+    SqlNode stmt;
+    SqlNodeList sqlOptions = SqlNodeList.EMPTY;
+}
+{
+    <ANALYZE>
+    [
+        <WITH> <OPTIONS>
```

Comment:
I think `WITH OPTIONS` might be more natural at the end of the query, but I do not have a strong opinion here. Maybe we could allow them before and after? I imagine that `WITH OPTIONS` after the query might create some challenges during parsing.

Reply:
for `CREATE JOB` the options are before query so here we can keep it the same way